openshift/rh-che.config.yaml

```diff
@@ -39,6 +39,7 @@ data:
   CHE_SYSTEM_SUPER__PRIVILEGED__MODE: "true"
   CHE_WEBSOCKET_ENDPOINT: "wss://che.prod-preview.openshift.io/api/websocket"
   CHE_WEBSOCKET_ENDPOINT__MINOR: "wss://che.prod-preview.openshift.io/api/websocket-minor"
+  CHE_WORKSPACE_STORAGE_PREFERRED__TYPE: "ephemeral"
```

Comment:
Should it replace https://github.com/redhat-developer/rh-che/pull/1982/files#diff-89824f4a1e5e379f4f8378527ac0aab8R44 ?

Reply:
I think so, but need to check with @akurinnoy

Reply:
yeah, as I understand `CHE_WORKSPACE_PERSIST__VOLUMES_DEFAULT` is deprecated and does not work anymore (worth removing it from upstream also)

Reply:
`CHE_WORKSPACE_PERSIST__VOLUMES_DEFAULT` can be safely removed as UD doesn't rely on it anymore.

Reply:
PR updated